code/src/index.css

```diff
@@ -5,9 +5,154 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-image: url("/roses-background.jpg");
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h1 {
+  font-size: 40px;
+  margin-top: 100px;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-evenly;
+  height: 400px;
+  top: 0px;
+  text-align: center;
+  background: darkslategray;
+  color: white;
+}
+
+.circle-1 {
+  height: 40px;
+  width: 300px;
+  background: rgb(0, 38, 255);
+  border-radius: 100%;
+}
+
+.circle {
+  position: absolute;
+  top: 30px;
+  right: 250px;
+  height: 70px;
+  width: 70px;
+  background: cyan;
+  border-radius: 100%;
+}
+
+.last-circle {
+  position: absolute;
+  top: 30px;
+  left: 250px;
+  height: 70px;
+  width: 70px;
+  background: cyan;
+  border-radius: 100%;
+}
+
+.big-circle {
+  position: absolute;
+  height: 150px;
+  width: 150px;
+  background: lightpink;
+  border-radius: 100%;
+  top: 175px;
+  right: 20px;
+}
+
+.another-circle {
+  position: absolute;
+  height: 150px;
+  width: 150px;
+  background: lightpink;
+  border-radius: 100%;
+  top: 175px;
+  left: 20px;
+}
+
+.form {
+  border: 2px solid darkslategray;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: column;
+  padding: 10px;
+  text-align: center;
+  background: lightpink;
+  height: 500px;
+  width: 400px;
+  border: 1px solid lightslategray;
+
+  margin: 30px;
+}
+
+.form-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  align-self: center;
+}
+
+.time-container {
+  padding: 10px;
+  margin: 10px;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.genre-container {
+  padding: 10px;
+  margin: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.experience {
+  padding: 10px;
+  margin: 10px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.button {
+  width: 70px;
+  height: 40px;
+  padding: 5px;
+  margin: 5px;
+  border: 2px solid black;
+  border-radius: 10px;
+  background: cyan;
+}
+
+.button-container {
+  display: flex;
+  align-self: center;
+}
+
+.footer-container {
+  text-align: start;
+  background: darkslategray;
+  color: white;
+  height: 250px;
+  bottom: 0px;
+  display: flex;
+  flex-direction: column;
+}
+
+.footer-text {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-top: 80px;
+  font-size: 20px;
```

Comment:
If you have more time to put in this project maybe add some media queries to make it responsive in mobile view.